tools/bbbconfgen/test/bitcoin-template.conf

```diff
@@ -3,6 +3,7 @@ mainnet={{ bitcoind:mainnet }}
 testnet={{ bitcoind:testnet #default: 0 }}
 rpcconnect={{ bitcoind:rpcconnect }}
 dbcache={{ bitcoind:dbcache #default: 300 }}
-seednode={{ bitcoind:seednode:1 #rmLine }}
-seednode={{ bitcoind:seednode:2 #rmLine }}
-seednode={{ bitcoind:seednode:3 #rmLine }}
+{{ bitcoind:testnet #check }}printtoconsole=1                                                    
```

Comment:
Switch it around i.e.: `printtoconsole=1 {{ bitcoind:testnet #check }}`. Easier to read and consistent with the `seednode` entries below.

Reply:
I wanted to explicitly have both options present as an example. But I agree, in the new config files it's like you suggest always, so maybe that's not necessary.

Reply:
adjusted to your suggestion